src/redesign/_style.scss

```diff
@@ -646,7 +646,10 @@ select.form-control {
     }
 }
 .password-visibility{
-  .btn-icon__icon-container .btn-icon__icon {
+  height: 1.5rem !important;
+  width: 1.5rem !important;
+  color: $gray-500;
+  &:hover {
```

Comment:
At hover password icon would be white and icon button should be gray. Can you please double check it again? Its only setting gray color on hover.